tests/utils/channel_utils.test.jsx

```diff
@@ -53,4 +61,94 @@ describe('Channel Utils', () => {
+                });
+            });
+
+            it('if there is no current user', () => {
+                mockedCurrentUser = null;
+
+                const channel = {
+                    type: Constants.GM_CHANNEL,
+                    display_name: 'Display Name, with a comma!',
+                };
+                expect(Utils.getChannelDisplayName(channel)).toEqual(channel.display_name);
+            });
+        });
+
+        describe('should splice out current username with three users', () => {
+            const expectedChannelName = 'jason-4, jason-admin';
+
+            beforeEach(() => {
+                mockedCurrentUser = {username: 'jason'};
+            });
+
+            it('when at the beginning', () => {
+                const channel = {
+                    type: Constants.GM_CHANNEL,
+                    display_name: 'jason, jason-4, jason-admin',
+                };
+                expect(Utils.getChannelDisplayName(channel)).toEqual(expectedChannelName);
+            });
+
+            it('when in the middle', () => {
+                const channel = {
+                    type: Constants.GM_CHANNEL,
+                    display_name: 'jason-4, jason, jason-admin',
+                };
+                expect(Utils.getChannelDisplayName(channel)).toEqual(expectedChannelName);
+            });
+
+            it('when at the end', () => {
+                const channel = {
+                    type: Constants.GM_CHANNEL,
+                    display_name: 'jason-4, jason-admin, jason',
+                };
+                expect(Utils.getChannelDisplayName(channel)).toEqual(expectedChannelName);
+            });
+        });
+
+        describe('should splice out current username with four users', () => {
+            const expectedChannelName = 'test, test2, test3';
+
+            beforeEach(() => {
+                mockedCurrentUser = {username: 'test1'};
+            });
+
+            it('when at the beginning', () => {
+                const channel = {
+                    type: Constants.GM_CHANNEL,
+                    display_name: 'test1, test, test2, test3',
+                };
+                expect(Utils.getChannelDisplayName(channel)).toEqual(expectedChannelName);
+            });
+
+            it('when in the middle', () => {
+                const channel = {
+                    type: Constants.GM_CHANNEL,
+                    display_name: 'test, test2, test1, test3',
+                };
+                expect(Utils.getChannelDisplayName(channel)).toEqual(expectedChannelName);
+            });
+
+            it('when at the end', () => {
+                const channel = {
+                    type: Constants.GM_CHANNEL,
+                    display_name: 'test, test2, test3, test1',
+                };
+                expect(Utils.getChannelDisplayName(channel)).toEqual(expectedChannelName);
+            });
+        });
+    });
```

Comment:
Can we have a test case where the resulting displayname has suffix of `...` (internally it exceeded `MAX_CHANNEL_NAME_LENGTH`)?

Reply:
Yeah, I meant to add test cases for this, but honestly I have no idea why this block is even here. If the string is too long, it just adds `...` ..., but without truncating? So now it's longer, and nothing is actually being ellipsized.

Everywhere I look, I think we're using CSS to ellipsize, so I'm inclined to just rip this block out as I can't think of way it was doing something useful before.